### PR TITLE
Mercury 13.8.0 plat 8313

### PR DESCRIPTION
**Pull request by noam-arad**

### CODE REVIEW

File: alpha/lib/model/DeliveryProfileLive.php [hunk 1]
```diff
@@ -324,8 +324,12 @@ protected function getLivePackagerUrl($serverNode, $streamFormat = null)
 		
 		$livePackagerUrl = "$livePackagerUrl/p/$partnerID/e/$entryId/sd/$segmentDuration/";
 		$secureToken = $this->generateLiveSecuredPackagerToken($livePackagerUrl);
-		$livePackagerUrl .= "t/$secureToken/"; 
-		
+		$livePackagerUrl .= "t/$secureToken/";
+
+		$entry = $this->getDynamicAttributes()->getEntry();
+		if ($entry->getExplicitLive() &&  (!$entry->canViewExplicitLive()))
```
Comment (yossipapi) [inline alpha/lib/model/DeliveryProfileLive.php]: @kalturaguy will this require changes on the packager configuration file as well?

As to passing "type/user" it would be better to always pass type if explicitLive is turned on rather than passing it only if its type user. So if user pass user and if admin pass admin.

Aslo maske "user" & "admin" const's.

File: deployment/updates/scripts/xml/notifications/explicit_live_notification.template.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
 				<eventConditions objectType="array">
 					<item objectType="KalturaEventObjectChangedCondition">
 						<description>Entry view mode or recording status changes</description>
-						<modifiedColumns>{view_mode,recording_status}</modifiedColumns>
+						<modifiedColumns>view_mode,recording_status</modifiedColumns>
```
Comment (yossipapi) [inline deployment/updates/scripts/xml/notifications/explicit_live_notification.template.xml]: Did you meant to merge this change along with the url modification? 
It would be better to separate to 2 different pull requests.

Comment (noam-arad) [reply deployment/updates/scripts/xml/notifications/explicit_live_notification.template.xml]: you are correct, will do

File: alpha/lib/model/DeliveryProfileLive.php (hunk 1 above)
Comment (kalturaguy) [reply alpha/lib/model/DeliveryProfileLive.php]: @yossipapi  yes, we are modifying the packager config too...

Comment (noam-arad) [reply alpha/lib/model/DeliveryProfileLive.php]: I looked at the liveController code, since we only set a clipTo value if explicit live is enabled then even if we pass /type/user when it is off it won't have any effect.
I will make the requested change.
